### PR TITLE
Identity Federation, closes MISC-267

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: content/tutorials/getting-started/index.md [hunk 1]
```diff
@@ -17,16 +17,14 @@ In this tutorial, we'll describe every step to help you run your first test with
...
 * A Java Development Kit (JDK) installed in version 8 or 11
 {{< /alert >}}
 
 {{< alert tip >}}
 **Recommendations**
 
-* The [Gatling bundle](https://gatling.io/open-source/#downloadgatling) is beginner friendly and is the best choice if you're just getting started.
-* For long term projects, consider using a Gradle, Maven or SBT Gatling project. You can download some Gatling Enterprise ready sample projects from the Cloud dashboard directly!
+* The [Gatling bundle](https://gatling.io/open-source/#downloadgatling) is beginner-friendly and is the best choice if you're just getting started.
+* For long term projects, consider using a **Gradle**, **Maven** or **SBT** Gatling project. You can download some Gatling Enterprise ready sample projects from the Cloud dashboard directly!
```
Comment (guilgaly) [inline content/tutorials/getting-started/index.md:27]:
```suggestion
* For long term projects, consider using a **Gradle**, **Maven** or **SBT** Gatling project. You can download some sample projects ready for Gatling Enterprise directly from the Cloud dashboard!
```
(or "Gatling Enterprise-ready sample projects", but that's a bit of a moutfull)

File: content/tutorials/getting-started/index.md [hunk 2]
```diff
@@ -46,21 +44,19 @@ Before we start, make sure to go to [cloud.gatling.io](https://cloud.gatling.io)
 
 Gatling Enterprise handles users by putting them in organizations. This will allow you to work with multiple people on the same test projects, so they can either run new tests or view the results of previous ones.
 
-A user can be part of either a team within an organization or have access to the organization as a whole. In all cases, a user must be part of an organization. Hence, once signed up and connected, you'll need to create one:
-
-**If you are the first person in your organization to join Gatling, sign up and create an account**. In this case, you will need to create an organization:
+A user can be part of either a team within an organization or have access to the organization as a whole.
```
Comment (guilgaly) [inline content/tutorials/getting-started/index.md:47]:
```suggestion
A user can either be part of a team within an organization, or have access to that organization as a whole. 
```

File: content/reference/admin/users/index.md [hunk 3]
```diff
@@ -13,13 +13,12 @@ To access the Users administration, click on the {{< icon user >}} icon in the n
 
 {{< img src="users-table.png" alt="Users table" >}}
 
-To invite a user to your organization, click on the **Create** button and fill in the user information:
+To invite a user to your organization, click on the **Invite** button and fill user information:
```
Comment (ccedric) [inline content/reference/admin/users/index.md:16]: I'd keep "fill in"

File: content/reference/admin/users/index.md [hunk 4]
```diff
@@ -45,3 +44,9 @@ There are 4 different user roles in Gatling Enterprise:
 | Administrate API Tokens, Users and Teams |                    |                    |                    | Own team           |
 
 Each role can be global or team-specific.
+
+An invitation email will be sent to the users.
```
Comment (ccedric) [inline content/reference/admin/users/index.md:48]: Repetitive?
We already said that earlier.

File: content/reference/user/login/index.md [hunk 5]
```diff
@@ -6,13 +6,31 @@ lastmod: 2021-08-05T13:13:30+00:00
 weight: 10010
 ---
 
-Thanks so much for trying Gatling Cloud. To access Gatling Cloud you can sign up for a free trial plan here or visit https://cloud.gatling.io/ to log in.
+Thanks so much for trying Gatling Cloud. To access Gatling Cloud visit [https://cloud.gatling.io/](https://cloud.gatling.io/) to log in.
```
[the remark this answers is not in the document]
Comment (ccedric) [reply content/reference/user/login/index.md:9]: Gatling Enterprise Cloud

File: content/reference/user/login/index.md [hunk 6]
```diff
@@ -6,13 +6,31 @@ lastmod: 2021-08-05T13:13:30+00:00
...
+#### Create your own account
 
-{{< img src="login-github.png" alt="Login button" >}}
+{{< img src="login-register.png" alt="Login register" >}}
+
+Clicking on the **Register** link will redirect you at the sign up form to create your account.
+
+{{< img src="login-signup.png" alt="Login Signup" >}}
+
+Fill the form and press **Register** to create your account and receive an invitation email.
```
[the remark this answers is not in the document]
Comment (ccedric) [reply content/reference/user/login/index.md:21]: ~invitation~ validation

File: content/reference/user/organization/index.md [hunk 7]
```diff
@@ -6,13 +6,20 @@ lastmod: 2021-08-05T13:13:30+00:00
...
 
-{{< img src="organization-page-link.png" alt="Organization page link" >}}
+{{< img src="menu.png" alt="Organization settings" >}}
 
-## Profile information
+## Switch between multiple organizations
 
-{{< img src="organization-profile.png" alt="Organization profile information" >}}
+If you've been invited to multiple organizations, 
+you can switch between them by clicking on the organization name in top right corner dropdown.
```
Comment (ccedric) [inline content/reference/user/organization/index.md:16]: in the top right corner

File: content/reference/user/organization/index.md [hunk 8]
```diff
@@ -25,48 +32,63 @@ Click on the pen icon to edit the **Organization name**.
...
+{{< /alert >}}
+
+{{< img src="invitations.png" alt="Invitations" >}}
+
+* **Email** - Recipient of the invitation.
+* **Sent** - Date the invitation was sent.
+* **Expire** - End date of the invitation.
+
+
+
```
Comment (ccedric) [inline content/reference/user/organization/index.md:94]: remove the empty lines

File: content/reference/user/organization/index.md [hunk 9]
```diff
@@ -25,48 +32,63 @@ Click on the pen icon to edit the **Organization name**.
...
+{{< alert warning >}}
+This section is only available to the System Admin.
+{{< /alert >}}
+
+{{< img src="invitations.png" alt="Invitations" >}}
+
+* **Email** - Recipient of the invitation.
+* **Sent** - Date the invitation was sent.
+* **Expire** - End date of the invitation.
```
Comment (ccedric) [inline content/reference/user/organization/index.md:91]: After this date, the invitation will no longer be valid, and you"ll need to create manually a new invitation

File: content/reference/user/organization/index.md [hunk 10]
```diff
@@ -25,48 +32,63 @@ Click on the pen icon to edit the **Organization name**.
...
+## Invitations
+
+{{< alert warning >}}
+This section is only available to the System Admin.
+{{< /alert >}}
+
+{{< img src="invitations.png" alt="Invitations" >}}
+
+* **Email** - Recipient of the invitation.
+* **Sent** - Date the invitation was sent.
```
Comment (ccedric) [inline content/reference/user/organization/index.md:90]: The wording seems strange to me

File: content/reference/user/login/index.md [hunk 11]
```diff
@@ -24,23 +42,12 @@ You will need to **authorize** it. We only require access to your public profile
...
 
-{{< img src="organization-creation.png" alt="OAuth GitHub page" >}}
+{{< img src="organization-creation.png" alt="Organization creation page" >}}
 
 {{< alert warning >}}
-You shouldn't create an organization if you want to join an existing one.
-{{< /alert >}}
-
-{{< alert danger >}}
 Creating an organization is irreversible. If you want to leave your organization, please contact [support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8).
```
Comment (ccedric) [inline content/reference/user/login/index.md:52]: If you want to destroy your organization.
Users can leave the organization (if he is not the last global system admin)

File: content/reference/user/organization/index.md (hunk 10 above)
Comment (franciscotco) [reply content/reference/user/organization/index.md:90]: suggestion: `Date where the invitation was sent.`

File: content/reference/user/organization/index.md [hunk 12]
```diff
@@ -25,48 +32,60 @@ Click on the pen icon to edit the **Organization name**.
...
+
+{{< alert warning >}}
+This section is only available to the System Admin.
+{{< /alert >}}
+
+{{< img src="invitations.png" alt="Invitations" >}}
+
+* **Email** - Recipient of the invitation.
+* **Sent** - Date where the invitation was sent.
+* **Expire** - After this date, the invitation will no longer be valid, and you"ll need to generate a new invitation.
```
Comment (guilgaly) [inline content/reference/user/organization/index.md:91]:
```suggestion
* **Expire** - After this date, the invitation will no longer be valid, and you'll need to generate a new invitation.
```

**Review by guilgaly — APPROVED** [review]
A small typo + still an old comment remaining to be addressed, OK otherwise.

**Pull request merged**